src/pages/preferences/fieldMaps.ts

```diff
@@ -0,0 +1,173 @@
+    value: "UTC",
+  },
+  {
+    str: "American Samoa, Niue",
+    value: "Pacific/Niue",
+  },
+  {
+    str: "Hawaii",
+    value: "Pacific/Tahiti",
+  },
+  {
+    str: "Marquesas Islands",
+    value: "Pacific/Marquesas",
+  },
+  {
+    str: "Alaska",
+    value: "America/Anchorage",
+  },
+  {
+    str: "Pacific Time",
+    value: "America/Vancouver",
+  },
+  {
+    str: "Mountain Time",
+    value: "America/Denver",
+  },
+  {
+    str: "Central Time",
+    value: "America/Chicago",
+  },
+  {
+    str: "Eastern Time",
+    value: "America/New_York",
+  },
+  {
+    str: "Venezuela",
+    value: "America/Caracas",
+  },
+  {
+    str: "Atlantic Time",
+    value: "America/Barbados",
+  },
+  {
+    str: "Newfoundland",
+    value: "America/St_Johns",
+  },
+  {
+    str: "Argentina, Paraguay",
+    value: "America/Belem",
+  },
+  {
+    str: "Fernando de Noronha",
+    value: "America/Noronha",
+  },
+  {
+    str: "Cape Verde",
+    value: "Atlantic/Cape_Verde",
+  },
+  {
+    str: "Iceland",
+    value: "Atlantic/Reykjavik",
+  },
+  {
+    str: "United Kingdom, Ireland",
+    value: "Europe/London",
+  },
+  {
+    str: "Central European Time, Nigeria",
+    value: "Europe/Rome",
+  },
+  {
+    str: "Egypt, Israel, Romania",
+    value: "Europe/Bucharest",
+  },
+  {
+    str: "Ethiopia, Iraq, Yemen",
+    value: "Asia/Baghdad",
+  },
+  {
+    str: "Iran",
+    value: "Asia/Tehran",
+  },
+  {
+    str: "Dubai, Moscow",
+    value: "Europe/Moscow",
+  },
+  {
+    str: "Afghanistan",
+    value: "Asia/Kabul",
+  },
+  {
+    str: "Maldives, Pakistan",
+    value: "Antarctica/Davis",
+  },
+  {
+    str: "India, Sri Lanka",
+    value: "Asia/Kolkata",
+  },
+  {
+    str: "Nepal",
+    value: "Asia/Kathmandu",
+  },
+  {
+    str: "Bangladesh, Bhutan",
+    value: "Asia/Dhaka",
+  },
+  {
+    str: "Cocos Islands, Myanmar",
+    value: "Asia/Rangoon",
+  },
+  {
+    str: "Thailand, Vietnam",
+    value: "Asia/Bangkok",
+  },
+  {
+    str: "China, Hong Kong, Perth",
+    value: "Asia/Hong_Kong",
+  },
+  {
+    str: "Eucla (Unofficial)",
+    value: "Australia/Eucla",
+  },
+  {
+    str: "Japan, South Korea",
+    value: "Asia/Seoul",
+  },
+  {
+    str: "Australia Central Time",
+    value: "Australia/Adelaide",
+  },
+  {
+    str: "Australia Eastern Time",
+    value: "Australia/Sydney",
+  },
+  {
+    str: "Lord Howe Island",
+    value: "Australia/Lord_Howe",
+  },
+  {
+    str: "Russia Vladivostok Time",
+    value: "Asia/Vladivostok",
+  },
+  {
+    str: "Norfolk Island",
+    value: "Pacific/Norfolk",
+  },
+  {
+    str: "Fiji, Russia Magadan Time",
+    value: "Asia/Magadan",
+  },
+  {
+    str: "Chatham Islands",
+    value: "Pacific/Chatham",
+  },
+  {
+    str: "Tonga",
+    value: "Pacific/Tongatapu",
+  },
+  {
+    str: "Kiribati Line Islands",
+    value: "Pacific/Kiritimati",
+  },
+];
+
+export const awsRegions = [
+  {
+    str: "US-East-1",
+    value: "us-east-1",
+  },
```

Comment:
Could use some help finding where the rest of the regions are located inside of evergreen.

Reply:
i believe momentjs is used for all the timezones: https://momentjs.com/timezone/

Reply:
just realized you mean aws regions, not timezones

Reply:
you'll need to query for them (probably another ticket): https://github.com/evergreen-ci/evergreen/blob/master/service/user.go#L200